.eslintrc

```diff
@@ -20,10 +20,10 @@
   },
   "rules": {
 
-    // Strict mode
+    "_comment_1_": "Strict mode",
```

Comment:
JSON doesn't support JavaScript-style comments, so I use meaningless key name to write comments.

Reply:
I think `.eslintrc` is none of JS, JSON, and YML file. So it doesn't have to follow any syntax. 
But it would be better to remove the comments, I think they are useless.